src/Tree/types.ts

```diff
@@ -0,0 +1,7 @@
+export type NavigationElement = {
+  id: string;
+  name: string;
+  // important to have to compute if an element should be disabled
+  path: string;
```

Comment:
You already pass a isDisabled function no ? can this be removed ?

Reply:
Yes, but it's the data I send in the arguments of the function, so I need it ;)